src/EnergyPlus/SZVAVModel.hh

```diff
@@ -0,0 +1,129 @@
+// EnergyPlus, Copyright (c) 1996-2017, The Board of Trustees of the University of Illinois,
```

Comment:
The year needs to be updated to 2018 in this header file and the accompanying cc file.

Reply:
Has it been that long since I finished this? Will do.